ngraph/test/CMakeLists.txt

```diff
@@ -84,7 +84,6 @@ set(SRC
     op_eval/swish.cpp
     op_eval/strided_slice.cpp
     op_eval/variadic_split.cpp
-    op_is.cpp
```

Comment:
@iefode Can you please comment regarding what was wrong with the tests?  As far as I remember you have been working on these tests

Reply:
I don't remember the reason yet. Could you restore it?

Reply:
I suppose this test was considered useless since it relies on opset0, but anyway deletion from my point of view not required. I've rolled it back and removed some checks for deleted operations